homeassistant/components/fritz/__init__.py

```diff
@@ -63,6 +67,22 @@ def _async_unload(event):
     return True
 
 
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate config entry."""
+    _LOGGER.debug("Migrating from version %s", config_entry.version)
+
+    if config_entry.version == 1:
+        config_entry.version = 2
+        hass.config_entries.async_update_entry(
+            config_entry,
+            options={CONF_CONSIDER_HOME: DEFAULT_CONSIDER_HOME.total_seconds()},
+        )
+
+    _LOGGER.info("Migration to version %s successful", config_entry.version)
```

Comment:
This should be reverted. We don't need to bump a schema version to fix this.

Additionally, this will break the possibility for users to downgrade HA.

Instead, this should have been handled as default on the place used.

Reply:
https://github.com/home-assistant/core/blob/b528af75dec39a3320bcb944e326c630cf927fc6/homeassistant/components/fritz/common.py#L179

Reply:
Revert: <https://github.com/home-assistant/core/pull/51193>
Replacement: <https://github.com/home-assistant/core/pull/51194>

Reply:
> Additionally, this will break the possibility for users to downgrade HA.

oh yeah your are right ... honestly did not considered this 😕 
Many thanks for this hint 👍